spark/src/main/scala/org/apache/spark/sql/delta/coordinatedcommits/CommitCoordinatorClient.scala

```diff
@@ -237,11 +237,26 @@ object CommitCoordinatorProvider {
       name: String,
       conf: Map[String, String],
       spark: SparkSession): CommitCoordinatorClient = synchronized {
-    nameToBuilderMapping.get(name).map(_.build(spark, conf)).getOrElse {
+    getCommitCoordinatorClientOpt(name, conf, spark).getOrElse {
```

Comment:
Also update the documentation to say - throws the exception in case the give commit coordinator is not found.

Reply:
Updated